src/web/theme/modules/Footer/Footer.js

```diff
@@ -4,13 +4,43 @@ import StoreDetails from "../StoreDetails";
 import StoreContact from "../StoreContact";
 import EnhanceFooter from "./EnhanceFooter";
 import FooterQuery from "./FooterQuery.gql";
+import LoadingArea from "theme/ui/molecules/LoadingArea";
+import Logo from "theme/ui/atoms/Logo";
 import "./Footer.scss";
+import Mountain from "./Mountains.png";
 
-const Footer = ({ store }) => {
+const Footer = ({ loading, category }) => {
+  console.log(Mountain);
   return (
     <footer className="footer">
-      {store && store.owner && <StoreDetails owner={store.owner} />}
-      {store && store.phone && <StoreContact phone={store.phone} />}
+      <img className="mountain" src={Mountain} alt="mountain" />
+      <div className="footer__logo">
+        <Logo />
+        {loading ? (
+          <LoadingArea>Loading main category…</LoadingArea>
+        ) : (
+          <span
+            dangerouslySetInnerHTML={{
+              __html: category.description
+            }}
+          />
+        )}
+      </div>
+      <div className="footer__categories">
+        {loading ? (
+          <LoadingArea>Loading subCategories</LoadingArea>
```

Comment:
I don't think the mention "subCategories" is particularly useful here, a plain "Loading" would be better